src/tools/scripts/fixIdConseillersDansCollectionStatsTerritoires/fixIdConseillersDansCollectionStatsTerritoires.js

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env node
+'use strict';
+const { execute } = require('../../utils');
+
+const getTerritoires = async db => await db.collection('stats_Territoires').find().toArray();
+
+const getConseillerRecrute = async (db, id) => await db.collection('conseillers').findOne({ '_id': id, 'statut': 'RECRUTE' });
```

Comment:
```suggestion
const getConseillerRecrute = db => async id => await db.collection('conseillers').findOne({ '_id': id, 'statut': 'RECRUTE' });
```
Séparation des dépendances (db) et des paramètres de ta requête (id) pour faciliter les tests.

Reply:
Ok ! 
Par contre quid de faire des tests unitaire sur un script oneshot ? J'ai hésité à en mettre en place...

Reply:
Bon à savoir !

Reply:
Ce que j'aime bien faire c'est de montrer un exemple before de la logique executive sur un mini set de données mais je ne sais si c'est facile à faire ici

Reply:
@MorganP-projects Pas de souci pour les tests à mon avis c'est comme tu le sens, si tu penses que tu en as besoin pour te rassurer sur ton implémentation alors il faut que tu en fasse sinon ce n'est as forcément utile.
Je t'ai fais ces retours car j'ai remarqué que tu as extrait les implémentations d'accès aux données du reste du code, et cette façon de faire avec db => ... permet d'isoler encore plus en séparant l'outil de gestion de la bdd de la signature de la fonction et faire cela est utile de manière général quand tu fais des tests, mais ça ne t'oblige pas à en faire

Reply:
@romain-cambonie je regarde pour en faire un !
@marc-gavanier  je pense que je vais prendre l'habitude de traiter séparément les dépendances quand c'est possible (test ou non) !